docs/topics/api.rst

```diff
@@ -102,6 +106,9 @@ Sign-In
         A list of the user's pre-saved `Solitude payment method`_ objects.
         If the user hasn't saved any payment methods yet, this will be an
         empty list.
+    :>json string csrf_token: `Django CSRF`_ token string to protect against
+        cross site request forgery. You must include this in all subsequent
+        requests using the ``X-CSRFToken`` request header.
```

Comment:
As far as I can tell from reading the Django code, it goes like this:
* A response that renders a CSRF token also sets a cookie with that value
* A request to a protected view must pass in the token value. The value must match that of the cookie.

I think the idea is that an attacker on another domain does not have access to cookies so it cannot get the token. 

Because we don't have a landing page running the same Django code we cannot set the token value on page load. Instead, I made it so you get a token after successful login. This isn't bullet proof because it means an attacker could do a cross-domain post to the login view but for that to yield fruit they'd have to also steal an FxA token.